src/test/java/com/puppycrawl/tools/checkstyle/checks/coding/VariableDeclarationUsageDistanceCheckTest.java

```diff
@@ -323,15 +322,32 @@ public void testVariableDeclarationUsageDistanceSwitchExpressions2() throws Exce
     @Test
     public void testGeneral3Class() throws Exception {
         final String[] expected = {
-            "28:9: " + getCheckMessage(MSG_KEY, "a", 4, 1),
-            "46:9: " + getCheckMessage(MSG_KEY, "a", 2, 1),
-            "65:9: " + getCheckMessage(MSG_KEY, "a", 4, 1),
-            "106:9: " + getCheckMessage(MSG_KEY, "m", 4, 1),
+            "45:9: " + getCheckMessage(MSG_KEY, "a", 2, 1),
         };
 
         verifyWithInlineConfigParser(
-                getPath("InputVariableDeclarationUsageDistanceGeneral3.java"),
-                expected);
+                getPath("InputVariableDeclarationUsageDistanceGeneral3.java"), expected);
+    }
+
+    @Test
+    public void testGeneral4Class() throws Exception {
```

Comment:
`testGeneralClass4`, please change it at other places in the test file :)

Reply:
Done. Thanks for pointing this out.

Reply:
Resolved.